pyomo/contrib/trustregion/examples/example1.py

```diff
@@ -2,12 +2,23 @@
 #
 #  Pyomo: Python Optimization Modeling Objects
 #  Copyright 2017 National Technology and Engineering Solutions of Sandia, LLC
-#  Under the terms of Contract DE-NA0003525 with National Technology and 
-#  Engineering Solutions of Sandia, LLC, the U.S. Government retains certain 
+#  Under the terms of Contract DE-NA0003525 with National Technology and
+#  Engineering Solutions of Sandia, LLC, the U.S. Government retains certain
 #  rights in this software.
+#
+#  The Institute for the Design of Advanced Energy Systems Integrated Platform
+#  Framework (IDAES IP) was produced under the DOE Institute for the
+#  Design of Advanced Energy Systems (IDAES), and is copyright (c) 2018-2021
+#  by the software owners: The Regents of the University of California, through
+#  Lawrence Berkeley National Laboratory,  National Technology & Engineering
+#  Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia University
+#  Research Corporation, et al.  All rights reserved.
+#
 #  This software is distributed under the 3-clause BSD License.
 #  ___________________________________________________________________________
 
+# This model is an adaptation of Eason & Biegler's original example in the
+# previous version of the Trust Region solver.
```

Comment:
Would you add the full citation to the paper that this example came from to make it easier to find?